packages/dnb-eufemia/src/extensions/payment-card/icons/CardInMedium.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+const CardInMedium = (props) => (
```

Comment:
```suggestion
const CardIn = (props) => (
```

We could perhaps rename/replace all references of `CardInMedium` with `CardIn` 🤔 
I think this fits better with the existing icons in PaymentCard, as they don't seem to be suffixed with the icon size(like medium).